### ISSUE TITLE
Login and registration forms are missing autocomplete (WCAG 2.1 SC 1.3.5, level AA)

### ISSUE THREAD

**Issue by eatyourgreens** [zooniverse/Panoptes-Front-End#6944]
## Expected behavior
WCAG 2.1 requires autocomplete on text inputs for user account info, to meet [SC 1.3.5: Identify input purpose](https://www.w3.org/WAI/WCAG21/Understanding/identify-input-purpose.html).

## Current behavior
None of the text inputs have autocomplete eg. `autocomplete='email'`.
